ui/app/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -0,0 +1,181 @@
+import React, { useContext, useMemo, useRef, useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { I18nContext } from '../../../contexts/i18n';
+import { useEqualityCheck } from '../../../hooks/useEqualityCheck';
+import Button from '../../ui/button';
+import Popover from '../../ui/popover';
+import { updateViewedNotifications } from '../../../store/actions';
+import { getTranslatedUINoficiations } from '../../../../../shared/notifications';
+import { getSortedNotificationsToShow } from '../../../selectors';
+
+function getActionFunctions() {
+  const actionFunctions = {
+    1: () => {
+      global.platform.openTab({
+        url: 'https://metamask.io/download.html',
```

Comment:
Notification 1 has no action text, so this function isn't callable (there is no button).

Reply:
Removed in b4849b502